.ci-scripts/x86-64-unknown-linux-gnu-nightly.bash

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+API_KEY=$1
+if [[ ${API_KEY} == "" ]]
+then
+  echo "API_KEY needs to be supplied as first script argument."
+  exit 1
+fi
+
+# Compiler target parameters
+ARCH=x86-64
+PIC=true
+
+# Triple construction
+VENDOR=unknown
+OS=linux-gnu
+TRIPLE=${ARCH}-${VENDOR}-${OS}
+
+# Build parameters
+MAKE_PARALLELISM=4
```

Comment:
might need to make this adjustable for CircleCI. will see if this is "too much parallelism".